test/e2e/policies.test-suite/balloons/n4c16/test22-isolcpus/code.var.sh

```diff
@@ -0,0 +1,49 @@
+helm-terminate
+helm_config=${TEST_DIR}/balloons-isolcpus.cfg helm-launch balloons
```

Comment:
It does not make sense to launch the plugins here. Usually you don't have isolcpus set on the kernel command line at this point, which you later check and try to fix, once the policy is up and running. It make more sense to first check the kernel command line, fix it if there is a need, then reboot, verify the updated command line, and only after that install the policy.